app/assets/stylesheets/bootstrap_and_customization.css.scss

```diff
@@ -92,3 +145,24 @@ a:hover {
 .dropdown:hover .dropdown-menu {
   display: block;
 }
+
+.navbar-toggle {
+  margin-top: 22px;
+  margin-right: 25px;
+}
+.navbar-collapse {
+  background-color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`